setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages, setup
+
+requirements = ["pycryptodome>=3.0.0", "eth-abi>=4.0.0"]
+
+pydantic_requirements = ["pydantic>=2.0.0"]
+
+tests_requirements = ["pytest-cov>=4.0.0", "pytest>=7.0.0"]
+
+all_requirements = pydantic_requirements
+
+
+VERSION = "0.1.0"
+DESCRIPTION = "MerkleTree implementation"
+LONG_DESCRIPTION = "Yet another package for MerkleTree, this time compatible with OpenZeppelin implementation"
```

Comment:
Just a thought – it's often handy to have package metadata info right in the code, especially for debugging. Plus, I think it's a good habit.

For example, add a `__version__.py` file in the `merkle_zeppelin` package for this? It would look something like this:
```
VERSION = (0, 1, 0)

__name__ = '...'
__description__ = '...'
__version__ = '.'.join([str(i) for i in VERSION])
__author__ = 'Name Surname'
__author_email__ = 'engineer@email.com'
__url__ = 'https://company.com'
```

Then, you can load this info in the setup.py like this:
```
import pathlib
from setuptools import setup

here = pathlib.Path(__file__).parent.resolve()

# Load the package's metadata from __version__.py
# Avoid importing it directly to prevent loading external dependencies
# that setup.py will install later
about: dict[str, str] = {}
with (here / 'merkle_zeppelin' / '__version__.py').open() as f:
  exec(f.read(), about)

# Get the long description from the README file
long_description = (here / 'README.md').read_text(encoding='utf-8')
```

But again, totally your call.

Reply:
huh, that's huge pile of experience  "how to prepare package" - super super valuable, thanks!
ofc I will implement your proposition!